eZ/Publish/SPI/Limitation/Target/Builder/VersionBuilder.php

```diff
@@ -23,11 +24,20 @@ final class VersionBuilder
     /** @var array */
     private $targetVersionProperties = [];
 
+    private $updatedFields = [];
+
     public function build(): Target\Version
     {
         return new Target\Version($this->targetVersionProperties);
     }
 
+    public function updateFields(array $updatedFields): self
```

Comment:
are those instances of `Field` or are they actually `$updatedFieldIdentifiers`?

Reply:
`\eZ\Publish\API\Repository\Values\Content\Field[]`, docblock added.